src/pages/patientView/mutationalSignatures/MutationalSignatureBarChart.tsx

```diff
@@ -0,0 +1,346 @@
+import * as React from 'react';
+import {
+    VictoryBar,
+    VictoryAxis,
+    VictoryChart,
+    VictoryLabel,
+    VictoryTooltip,
+} from 'victory';
+import { action, computed } from 'mobx';
+import { observer } from 'mobx-react';
+import _ from 'lodash';
+import { IMutationalCounts } from 'shared/model/MutationalSignature';
+import {
+    getColorsForSignatures,
+    ColorMapProps,
+    colorMap,
+    IColorDataBar,
+    IColorLegend,
+    getLegendEntriesBarChart,
+    getxScalePoint,
+    LegendLabelsType,
+    DrawRectInfo,
+    LabelInfo,
+    getLengthLabelEntries,
+    createLegendLabelObjects,
+    formatLegendObjectsForRectangles,
+    getCenterPositionLabelEntries,
+} from './MutationalSignatureBarChartUtils';
+import { CBIOPORTAL_VICTORY_THEME } from 'cbioportal-frontend-commons';
+import { AxisScale } from 'react-mutation-mapper';
+import { scalePoint } from 'd3-scale';
+
+export interface IMutationalBarChartProps {
+    signature: string;
+    width: number;
+    height: number;
+    refStatus: boolean;
+    data: IMutationalCounts[];
+    version: string;
+    sample: string;
+    label: string;
+}
+
+const theme = _.cloneDeep(CBIOPORTAL_VICTORY_THEME);
+theme.legend.style.data = {
+    type: 'square',
+    size: 5,
+    strokeWidth: 0,
+    stroke: 'black',
+};
+
+export function formatLegendTopAxisPoints(data: IMutationalCounts[]) {
+    const constant_value = _.max(data.map(item => item.value)) || 0;
+    const groupedData = _.groupBy(getColorsForSignatures(data), 'group');
```

Comment:
using the string 'group' shortcut is not typesafe, so we trying to never do it.  better ```(g)=>g.group```

Reply:
Replaced this with ` (g)=>g.group`